core/domain/story_domain_test.py

```diff
@@ -232,13 +232,24 @@ def setUp(self):
         self.signup('user1@example.com', 'user1')
 
     def _assert_validation_error(self, expected_error_substring):
-        """Checks that the story passes validation."""
+        """Checks that the story passes validation.
+
+        Args:
+            expected_error_substring: string. String that should be a substring
```

Comment:
Can you change `string` to `str` in the parameter type? As this is the convention that is followed throughout the codebase.

Reply:
Done!